app/sessionStore.js

```diff
@@ -412,6 +412,7 @@ module.exports.defaultAppState = () => {
         guid: [],
         timestamp: 0
       }
-    }
+    },
+    menubar: {}
```

Comment:
The original intention for doing this was that there is only one system menu (and each Window would reference that).  But let me revisit this thought- thanks for bringing it up :smile: